compat/src/index.d.ts

```diff
@@ -21,7 +21,7 @@ declare namespace React {
 	export import StateUpdater = _hooks.StateUpdater;
 	export import useCallback = _hooks.useCallback;
 	export import useContext = _hooks.useContext;
-	export import useDebugValue = _hooks.useDebugValue;
```

Comment:
We probably don't wanna remove this

Reply:
Ooh that's not meant to be deleted. For that matter, I don't expect that this PR is necessarily "production grade"; it merely meets my needs.